pkg/bastionsession/bastionsession.go

```diff
@@ -35,24 +41,35 @@ func ChannelHandler(srv *ssh.Server, conn *gossh.ServerConn, newChan gossh.NewCh
 	if err != nil {
 		return err
 	}
-
+	user := conn.User()
 	// pipe everything
-	return pipe(lreqs, rreqs, lch, rch)
+	return pipe(lreqs, rreqs, lch, rch, config.Logs, user)
 }
 
-func pipe(lreqs, rreqs <-chan *gossh.Request, lch, rch gossh.Channel) error {
+func pipe(lreqs, rreqs <-chan *gossh.Request, lch, rch gossh.Channel, logs_location string, user string) error {
```

Comment:
I'm not 100% sure yet, but I think it's better to pass an `io.Writer` object so this function can stay generic and we can imagine replace the file logger by a `logstash` logger etc

Nothing to do right now :)